lib/reek/context/statement_counter.rb

```diff
@@ -0,0 +1,33 @@
+require_relative '../ast/node'
+require 'private_attr/everywhere'
+
+module Reek
+  module Context
+    # Responsible for counting the statements in a `CodeContext`.
+    class StatementCounter
+      attr_reader :value
+      private_attr_writer :value
+
+      def initialize
+        @value = 0
+      end
+
+      # :reek:DuplicateMethodCall: { max_calls: 2 }
```

Comment:
Technically it's correct, but we still shouldn't report DuplicateMethodCall here. Something we should fix in the future.